.travis.yml

```diff
@@ -0,0 +1,7 @@
+language: node_js
+node_js:
+  - "0.12"
```

Comment:
`jsdom` deprecated everything below `iojs` in one of its releases, we can't support both groups of versions simultaneously